src/Test/DepTyCheck/Gen.idr

```diff
@@ -396,48 +552,38 @@ namespace GenAlternatives
   relax : GenAlternatives True em a -> GenAlternatives ne em a
   relax = MkGenAlts . relaxT . unGenAlts
 
+  export %inline
+  strengthen : GenAlternatives ne em a -> Maybe $ GenAlternatives True em a
```

Comment:
Seemingly, the same comment as for `strengthen` for `Gen`s

Reply:
This `strengthen` has not been changed, it guarantees a non-empty list, not generators
https://github.com/buzden/deptycheck/blob/b7082a2a3a5d4b44ec7eea32c2c293221b431fc4/src/Test/DepTyCheck/Gen.idr#L285-L287